### ISSUE TITLE
How to connect lnd to zap or get the lndconnect scheme?

### ISSUE THREAD

**Issue by vwas2** [lightningnetwork/lnd#6488]
remote node runs debian lnd-linux-amd64-v0.14.3-beta.tar.gz

the scheme android asks for
`lndconnect://<host>:<port>?[cert=<base64url DER certifcate>&]macaroon=<base64url macaroon>` can be [qr generated](https://github.com/LN-Zap/lndconnect/blob/master/lnd_connect_uri.md#create-and-show-a-qr-code-using-command-line), but what is the host:port?   
the tls.cert and admin.maroon are in the lnd datadir.

**Comment by ziggie1984** [comment]
yes you can easily connect also your Node which is running behind tor, for zap-android it uses port 10009 (grpc) so it would look something like this:

`lndconnect://chjlq27t7igzmv66botvnec7lsic3syg7rkplgxq263wc3vbm4qj5ryd.onion:10009.....`

for more infos check this out:

https://github.com/rootzoll/raspiblitz/blob/v1.7/home.admin/config.scripts/bonus.lndconnect.sh#L76

or

https://github.com/LN-Zap/lndconnect/blob/master/lnd_connect_uri.md

**Comment by ziggie1984** [comment]
not sure which node setup your are running but you also need to create a hidden tor grpc service, so you can connect to it.

Because the above onion address, is the address of my hidden service which is listening on port 10009.
